aperturedb/ParallelQuery.py

```diff
@@ -30,16 +38,38 @@ def generate_batch(self, data):
 
         return q, blobs
 
+    def call_response_handler(self, r, b):
```

Comment:
Drew: 
"if hasattr( "response_handler") and callable( self.response_handler ):"